addons/sale/tests/test_payment_flow.py

```diff
@@ -639,3 +639,44 @@ def test_partial_payment_confirm_order(self):
             notification_mail_mock.assert_called_with(
                 self.env.ref('sale.mail_template_sale_confirmation'))
             self.assertEqual(self.sale_order.state, 'sale')
+
+    def test_automatic_invoice_author_uses_invoice_user_id(self):
+        """
+        Test that automatic invoice email author uses invoice_user_id instead of portal user
+        """
```

Comment:
This comment is not relevant anymore, as the mail would be sent by OdooBot.
You can drop it